lib/custom_a11y_rules.js

```diff
@@ -156,6 +171,26 @@ var customRules = {
             evaluate: function(node, options) {
                 return axe.utils.isHidden(node);
             },
-        }
+        },
+        {
+            'id': 'section-heading-first-child',
+            'metadata': {
+                'impact': 'minor',
```

Comment:
@benpatterson I made a [proposal to the Bok Choy repository](https://github.com/edx/bok-choy/pull/153) which might help here. Note that a failure is a failure even if we deem it minor. We can decide on a case-by-case basis if the PR should be able to be merged despite any "minor" failures.